src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
```

Comment:
What does the "type compatibility error" mean? Is it a warning about breaking binary compatibility of `OrientationHelper`? If so, that should not be an issue, since `OrientationHelper` is only used within Flowless.